Tests/UIKitTests/UIImageExtensionsTests.swift

```diff
@@ -130,6 +130,15 @@ final class UIImageExtensionsTests: XCTestCase {
         XCTAssertEqual(testImage.bytesSize, tintedImage.bytesSize)
     }
 
+    func testWithBackgroundColor() {
+        let size = CGSize(width: 1, height: 1)
+        let clearImage = UIImage(color: .clear, size: size)
+        let imageWithBackgroundColor = clearImage.withBackgroundColor(.black)
+        XCTAssertNotNil(imageWithBackgroundColor)
+        let blackImage = UIImage(color: .black, size: size)
+        XCTAssertEqual(imageWithBackgroundColor!.bytesSize, blackImage.bytesSize)
```

Comment:
Perhaps instead you should be comparing the PNG or JPEG data representation of the images? Just comparing the number of bytes doesn't seem accurate.

Reply:
Done